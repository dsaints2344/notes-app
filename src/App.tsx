import React from 'react';
import './App.css';
import { Divider } from 'antd';

class App extends React.Component<any, any>{
  public render() {
    return (
      <div>
        <div className="App">
          <h1 style={{ fontWeight: "bold" }}>Notes App</h1>
          <Divider />
        </div>
      </div>
    );
  }
}

export default App;
