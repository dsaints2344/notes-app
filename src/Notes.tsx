import * as React from 'react'
import {Card} from 'antd'
import INotesProps from './typings/INotesProps'
class Notes extends React.Component<INotesProps,any>{
    public render(){
        return(
            <div>
                <Card title={this.props.title}>
                    <p>{this.props.content}</p>
                </Card>
            </div>
        )
    }
}

export default Notes