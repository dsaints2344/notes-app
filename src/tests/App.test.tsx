import React from 'react';
import { render } from '@testing-library/react';
import App from '../App';
import Notes from '../Notes'
describe('Notes App tests', () =>{
  it('should render <Notes/> component', () => {
      const {container} = render(<Notes/>)
      expect(container.firstChild).toMatchSnapshot();
  })
})

