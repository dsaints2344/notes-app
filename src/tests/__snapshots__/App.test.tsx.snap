// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Notes App tests should render <Notes/> component 1`] = `
<div>
  Notes
</div>
`;
